### PR TITLE
🚸 Show more verbose error for signin fail

### PR DESCRIPTION
**Pull request by williamchong007**

### CODE REVIEW

File: components/dialogs/AuthDialog.vue [hunk 1]
```diff
@@ -410,7 +410,7 @@ export default {
           details: this.error.toString(),
         });
       }
-      return this.$t('AuthDialog.label.signInError');
+      return `${this.$t('AuthDialog.label.signInError')} - ${this.errorCode || ''}`;
```
Comment (nwingt) [inline components/dialogs/AuthDialog.vue:413]: `登入時發生錯誤，請在右下對話框與我們聯絡 -` 
Maybe show some text instead of just ''? For visual only.

Comment (williamchong007) [reply components/dialogs/AuthDialog.vue:413]: Not sure what would be a nice fallback, "unknown"?

Comment (nwingt) [reply components/dialogs/AuthDialog.vue:413]: Up to u ~

Comment (williamchong007) [reply components/dialogs/AuthDialog.vue:413]: How about
```
      let msg = this.$t('AuthDialog.label.signInError');
      if (this.errorCode) msg += ` - ${this.errorCode}`;
      return msg;
```

**Pull request merged**